rllib/connectors/tests/test_connector.py

```diff
@@ -0,0 +1,54 @@
+import unittest
+
+from ray.rllib.connectors.connector import Connector, ConnectorPipeline
+
+
+class TestConnectorPipeline(unittest.TestCase):
+    class Tom(Connector):
+        def to_config():
+            return "tom"
+
+    class Bob(Connector):
+        def to_config():
+            return "bob"
+
+    class Mary(Connector):
+        def to_config():
+            return "mary"
+
+    class MockConnectorPipeline(ConnectorPipeline):
```

Comment:
Can we write a pipeline test with existing connectors that is meaningful?

Reply:
actually, this is specifically for testing the append/insert/remove functionalities of the base ConnectorPipeline class.
there are pipeline tests in test_agent.py and test_action.py that test with existing connectors.